lib/homepage_helper.rb

```diff
@@ -4,6 +4,16 @@ class HomepageHelper
   def self.resolve(request = nil, current_user = nil)
     return "custom" if ThemeModifierHelper.new(request: request).custom_homepage
 
+    enabled = false
```

Comment:
I don’t think we need to set `enabled` to `false`, do we? :thinking:

Reply:
we don't really need but it's more explicit